packages/barcode-scanning/src/definitions.ts

```diff
@@ -229,6 +229,14 @@ export interface StartScanOptions {
    * @default Resolution['1280x720']
    */
   resolution?: Resolution;
+  /**
+   * Allow camera usage on iPad while in multitasking mode.
+   *
+   * Only available on iOS. Defaults to false.
+   *
+   * @since 7.5.0
```

Comment:
```suggestion
   * Only available on iOS (16.0+).
   *
   * @since 7.5.0
   * @default false
```

Do not forgot to run `npm run docgen` after that to update the README.md.